test/errors-specs.js

```diff
@@ -125,4 +125,11 @@ describe('errorFromCode', () => {
       });
     }
   }
+  it('should throw unknonw error for unknown code', () => {
```

Comment:
for some unknown reasons ... ;) thanks for pointing :+1: will correct that and merge it